Android/app/src/main/res/values/strings.xml

```diff
@@ -272,7 +272,7 @@
     <string name="text_clock">World Clock</string>
     <string name="select_timezone">Select Timezone</string>
     <string name="search_timezone">Search Timezone..</string>
-
+    <string name="time_zone">Asia/Kolkata</string>
```

Comment:
we already have this string in strings.xml as sample_timezone
why another string with same value?